public/samples/on-chain-gas-estimate.sol

```diff
@@ -22,25 +36,28 @@ contract SenderReceiver is AxelarExecutable {
         string calldata message_
     ) external payable {
         bytes memory payload = abi.encode(message_);
-        
-        uint256 estimate = estimator.estimateGasFee(destinationChain, destinationAddress, payload, 200000, '');
 
+        uint256 estimate = getEstimate(
```

Comment:
I think this estimation part is supposed to happen off-chain, not in the contract. We still need to attach the right amount of value to the TX that is equal to estimate.